### ISSUE TITLE
Rename AZD Templates

### ISSUE THREAD

**Issue by gkulin** [Azure/azure-dev#2073]

**Comment by savannahostrowski** [comment]
@rajeshkamal5050 This should be done ahead of 5/9.

We just need to update the titles and paragraphs in our ToDo template READMEs + the repo side rail.

For example:
Title: 
![image](https://user-images.githubusercontent.com/8949415/236059481-1bf95666-6954-4729-9ea8-58f1fa19ce4a.png)

First paragraph:
![image](https://user-images.githubusercontent.com/8949415/236059510-e59fa42d-527b-4d31-ac17-23a26d299838.png)

Siderail:
![image](https://user-images.githubusercontent.com/8949415/236059528-1c678efa-2e1a-402f-bb6e-e51ff7b8c217.png)


Once this is done, we'll update awesome-azd

**Comment by gkulin** [comment]
@hemarina thanks for pointing that out, yes we do!

Let's go with this: 
Current Name | Proposed Name | Proposed Short Description
-- | -- | --
ToDo Application with a Java API and Azure Database for PostgreSQL - Flexible Server on Azure App Service | React Web App with Java API and PostgreSQL | A blueprint for getting a React web app with a Java API and a PostgreSQL database on Azure. The blueprint includes sample application code (a ToDo web app) which can be removed and replaced with your own application code. Add your own source code and leverage the Infrastructure as Code assets (written in Bicep) to get up and running quickly.

**Comment by jongio** [comment]
1. I recommend including Azure in the name because it will appear in Google search results and helps with SEO to have it there.
2.  We'll also need a way to differentiate the samples.  I noticed above that you have the same name for multiple sames in the title.  For SEO reasons we shouldn't duplicate the title.  Imagine searching for "react web app azure" and seeing a number of results with the same title.  The choice on which one to click on should be apparent in the Title.
3. I recommend just removing "ToDo APplication" from existing titles, if your primary driver is length.

**Comment by savannahostrowski** [comment]
The primary driver isn't just length. It's about making it more approachable for application developers to identify that a template might be relevant to their local application stack, since they may not know about Azure offerings. These names were a joint decision with Starshot folks, LT, and azd. I can provide more details if you're interested via DM

**Comment by jongio** [comment]
Ultimately I recommend creating unique names for each template so they are distinguishable via title only.  I'd also include Azure in the title because they are Azure templates.

**Comment by gkulin** [comment]
@jongio @savannahostrowski , I just updated the table at the top of this issue. if those changes look good (adding "on Azure
 to the end of each & distinguishing the duplicates), I'll go ahead and open a PR to update them

**Comment by savannahostrowski** [comment]
LGTM - let's get them updated!

**Comment by savannahostrowski** [comment]
@weikanglim Can we update the names in the template list/init to reflect the unique name updates above? No need to add on Azure in the commands because that's sort of obvious based on context.

**Comment by savannahostrowski** [comment]
Yeah - we'll need to make another small PR to do a couple last minute amendments.

**Comment by weikanglim** [comment]
@savannahostrowski

1. Do we need "on Azure" when the user is initializing from a template? They're already in the context of `azd`.
2. The template duplication was fixed, we added `- Terraform` for the TF specific one here[https://github.com/Azure/azure-dev/blob/2bab4461e3face1688ef5afc296d963108636715/cli/azd/resources/templates.json#L43](url)

**Comment by savannahostrowski** [comment]
No need for "on Azure" outside of the template title on GitHub.
